### ISSUE TITLE
No logs published to Cloudwatch

### ISSUE THREAD

**Issue by ggrzesiuk** [noxdafox/rabbitmq-cloudwatch-exporter#40]
Hi, 

it seems there is an issue with publishing logs to cloudwatch. 
My cofiguration it as follows: 
- RabbitMQ 3.9.8
- Erlang 24.1
- cloudwatch_exporter 1.0.1

my rabbitmq.conf 

cloudwatch_exporter.aws.region = eu-west-1
cloudwatch_exporter.metrics.overview.enable = true
cloudwatch_exporter.metrics.vhost.enable = true
cloudwatch_exporter.metrics.node.enable = true
cloudwatch_exporter.metrics.exchange.enable = true
cloudwatch_exporter.metrics.queue.enable = true

advanced.config
[{lager,
  [{handlers, [{lager_cloudwatch_backend, [info, "RabbitMQ"]}]}]}].

env variable set :
AWS_DEFAULT_REGION=eu-west-1

Metrics are published correctly but there there are no logs in cloudwatch. 
I  attached the following policy to my role to publish metrics and logs:

{
    "Version": "2012-10-17",
    "Statement": [
      {
        "Effect": "Allow",
        "Action": [
          "cloudwatch:*",
          "logs:CreateLogGroup",
          "logs:CreateLogStream",
          "logs:PutLogEvents",
          "logs:DescribeLogStreams"
      ],
        "Resource": [
          "*"
      ]
    }
   ]
  }

There are no errors in rabbitmq related to cloudwatch exporter plugin (even in debug mode). 
Any suggestions what might prevent logs publishing to cloudwatch ? 
Thanks,
Greg

**Comment by noxdafox** [comment]
Thanks @Sonic0 but I believe the OP is having issues with logs and not metrics.

Metrics seem to work fine on his deployment too.

On my tests logs are being published as well.

**Comment by ggrzesiuk** [comment]
hi Guys, 

correct. Metrics are published just fine. I tried already with different policies including logs:* but it did not help. 
@Sonic0 and/or @noxdafox would you be able to help me to troubleshoot on my AWS account ? Kindly please let me know your availibility( today?) so we could try to solve it. 
Thanks in advance,
Greg

**Comment by Sonic0** [comment]
Hi @noxdafox @ggrzesiuk I'm sorry for the mistake.
I have replicated the @ggrzesiuk configuration with the same result: 
_There are no errors in rabbitmq related to cloudwatch exporter plugin (even in debug mode)._

During this day I will made other tests about this problem.

**Comment by ggrzesiuk** [comment]
hi @Sonic0, 
have you managed to troubleshoot this issue ?

**Comment by Sonic0** [comment]
> hi @Sonic0, have you managed to troubleshoot this issue ?

No, I haven't. I tried just an hour to troubleshoot this issue but nothing to do, no logs.

**Comment by ggrzesiuk** [comment]
@noxdafox and @Sonic0 would you mind if I schedule a session with you to try tro troubleshoot this issue? 
Maybe we are just missing something in configuration etc and it doesn't publish logs correctly. 
Your help would be very much appreciated. Tell whether today afternoon would work for you ? Anytime between 3 PM and 9 PM would work for me.

**Comment by ggrzesiuk** [comment]
@noxdafox, your help will be necessary here to troubleshoot this issue. Kindly please suggest suitable time to discuss or let me know whether I could provide some useful information on my setup/configuration.

**Comment by ggrzesiuk** [comment]
hi @noxdafox, 

would you be able to investigate that issue and try to provide a fix ?

**Comment by hvillar** [comment]
hi @noxdafox,
I have the same issue as @ggrzesiuk, is there some news about it ?

**Comment by noxdafox** [comment]
I finally managed to get to the bottom of the issue.

Since `v3.8.x`, RMQ switched from Lager to integrated Erlang logger to provide its logging facilities.

Hence, the current implementation does not work anymore. I will update the README and see if in future we can hook into the Erlang logger to provide the same functionality.
